Sources/App/Controllers/HomeController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Copyright © 2017 Patrick Balestra. All rights reserved.
+//
+
+import Vapor
+import HTTP
+
+final class HomeController {
+    
+    var droplet: Droplet!
+    
+    func addRoutes(droplet: Droplet) {
+        self.droplet = droplet
+        droplet.get(handler: self.home)
+        let home = droplet.grouped("home")
+        home.get(handler: self.home)
+    }
+    
+    func home(request: Request) throws -> ResponseRepresentable {
+        // Get all categories to display in sidebar
+        let categories = try Category.all()
+        
+        let accessories: [Accessory]
+        let featuredAccessory: Accessory
+        let featuredAccessoryImage: String
+        
+        // Limit of fetched items for accessories query
+        let visibleAccessoriesLimit = 20
+
+        // Get limited amout of accessoires sorted by date
+        accessories = try Accessory.makeQuery().filter("approved", true).limit(visibleAccessoriesLimit).all().sorted(by: { $0.date.compare($1.date) == .orderedDescending })
+
+        // Get featured accessory from item id
+        let featuredAccessoryDbId = 26
```

Comment:
You can easily name this property `featuredAccessoryId` so we don't expose the fact it's stored in the database